### PR TITLE
[CTSKF 303] Add section 28 fee

### PR DESCRIPTION
**Pull request by jrmhaig**
#### What

Add a new miscellaneous fee for section 28 hearings

#### Ticket

[CCCD: Create Section 28 Fee](https://dsdmoj.atlassian.net/browse/CTSKF-303)

#### Why

From the 1 February 2023, Trial Advocate claims, with a representation order on or after 1 February 2023 AND a section 28 hearing on the case, are eligible for an additional £670 fixed fee.

#### How

Create a new fee scheme (14) to be identical to 13 but with an additional miscellaneous fee.

#### To do

- [x] Fix error message in API when attempting to add more than one s28 fee. Currently it refers to the unused materials fee.

### CODE REVIEW

**Review by NenehB — APPROVED** [review]
LGTM

File: db/seeds/schemas/add_agfs_fee_scheme_14.rb [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        Offence.joins(:fee_schemes).merge(FeeScheme.version(14)).merge(FeeScheme.agfs).distinct.count
+      end
+
+      def scheme_14_fee_type_count
+        Fee::BaseFeeType.agfs_scheme_14s.count
+      end
+
+      def create_agfs_scheme_fourteen
+        print "Finding AGFS scheme 13".yellow
+        # agfs_fee_scheme_thirteen = FeeScheme.find_by(name: 'AGFS', version: 13, start_date: Settings.clar_release_date.beginning_of_day)
```
Comment (mpw5) [inline db/seeds/schemas/add_agfs_fee_scheme_14.rb]: I guess this can be removed?

File: docs/fee_schemes.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+## Fee Schemes
+
+Fee schemes are managed by the `FeeScheme` model and are defined by;
+
+* **name**: Either `AGFS` or `LGFS`
+* **version**: The scheme number
+* **start_date**: Earliest representation order date
+* **end_date**: Last representation order date, or `nil`
+
+From LGFS fee scheme 10 and AGFS fee scheme 13 the schem is determined by the
```
Comment (mpw5) [inline docs/fee_schemes.md]: schem = scheme

Also I think this sentence could be clearer. Maybe "From LGFS fee scheme 10 and AGFS fee scheme 13 the main hearing date is used to determine the scheme in addition to the representation order date."?

**Review by mpw5 — APPROVED** [review]
Looks good. I've left a couple of comments about cosmetic issues but nothing that would block release.

**Review by mpw5 — APPROVED** [review]
👍

**Pull request merged**